osc-server-bom/bin/create-centos

```diff
@@ -60,6 +62,7 @@ find $BUILDDIR/var/lib/yum -type d -exec chmod 755 {} \;
 # 
 #
 install -o root -g root -m 644 /etc/resolv.conf $BUILDDIR/etc/
+install -o root -g root -m 777 /etc/profile.d/proxy.sh $BUILDDIR/etc/profile.d/
```

Comment:
As i understand it, install copies the file into the ovf image? if its not the case its fine.